boards/arm/efr32xg24_dk2601b/doc/index.rst

```diff
@@ -0,0 +1,154 @@
+
+The board configuration supports the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| MPU       | on-chip    | memory protection unit              |
++-----------+------------+-------------------------------------+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| COUNTER   | on-chip    | stimer                              |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | flash memory                        |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial                              |
++-----------+------------+-------------------------------------+
+| TRNG      | on-chip    | semailbox                           |
++-----------+------------+-------------------------------------+
+| WATCHDOG  | on-chip    | watchdog                            |
++-----------+------------+-------------------------------------+
+
+Other hardware features are currently not supported by the port.
+
+Connections and IOs
+===================
+
+In the following table, the column **Name** contains Pin names. For example, PA2
+means Pin number 2 on PORTA, as used in the board's datasheets and manuals.
+
++-------+-------------+-------------------------------------+
+| Name  | Function    | Usage                               |
++=======+=============+=====================================+
+| PA4   | GPIO        | LED0                                |
++-------+-------------+-------------------------------------+
+| PB0   | GPIO        | LED1                                |
++-------+-------------+-------------------------------------+
+| PB2   | GPIO        | Push Button 0                       |
++-------+-------------+-------------------------------------+
+| PB3   | GPIO        | Push Button 1                       |
++-------+-------------+-------------------------------------+
+| PA5   | USART0_TX   | UART Console EFM_BC_TX US0_TX       |
++-------+-------------+-------------------------------------+
+| PA6   | USART0_RX   | UART Console EFM_BC_RX US0_RX       |
++-------+-------------+-------------------------------------+
+
+The default configuration can be found in the defconfig file:
+
+	``boards/arm/efr32_radio/efr32_radio_brd4186a_defconfig``
+
+System Clock
+============
+
+The EFR32MG24 SoC is configured to use the 39 MHz external oscillator on the
+board.
+
+Serial Port
+===========
+
+The EFR32MG24 SoC has two USARTs.
```

Comment:
According to the EFR32xG24 Wireless SoC Reference Manual it has one USART and two EUSARTs.